smriprep/interfaces/fastsurfer.py

```diff
@@ -0,0 +1,673 @@
+        desc="Surface annotation files",
+        loc="label",
+        altkey="*annot"
+    )
+    aparc_ctab = OutputMultiPath(
+        File(exists=True),
+        loc="label",
+        altkey="aparc.annot.mapped.ctab",
+        desc="Aparc parcellation annotation ctab file",
+    )
+    mapped_024 = OutputMultiPath(
+        File(exists=True),
+        loc="label",
+        altkey="mapped*024",
+        desc="Mapped label files",
+    )
+    cortex = OutputMultiPath(
+        File(exists=True),
+        loc="label",
+        altkey="cortex",
+        desc="Cortex class label files",
+    )
+    aparc_aseg = OutputMultiPath(
+        File(exists=True),
+        loc="mri",
+        altkey="aparc?aseg",
+        desc="Aparc parcellation projected into aseg volume",
+    )
+    aparc_dkt_aseg = OutputMultiPath(
+        File(exists=True),
+        loc="mri",
+        altkey="aparc.DKTatlas*aseg*",
+        desc="Aparc parcellation from DKT atlas projected into aseg volume",
+    )
+    segment_dat = OutputMultiPath(
+        File(exists=True),
+        loc="mri",
+        altkey="segment_dat",
+        desc="Segmentation .dat files",
+    )
+    filled_pretess = OutputMultiPath(
+        File(exists=True),
+        loc="mri",
+        altkey="filled*pretess*",
+        desc="Pre-tessellation filled volume files",
+    )
+    sphere_reg = OutputMultiPath(
+        File(exists=True),
+        loc="surf",
+        altkey="sphere.reg",
+        desc="Spherical registration file",
+    )
+    preaparc = OutputMultiPath(
+        File(exists=True),
+        loc="surf",
+        altkey="preaparc",
+        desc="Pre-Aparc files",
+    )
+    aseg_stats = OutputMultiPath(
+        File(exists=True),
+        loc="stats",
+        altkey="aseg",
+        desc="Automated segmentation statistics file",
+    )
+    wmparc_stats = OutputMultiPath(
+        File(exists=True),
+        loc="stats",
+        altkey="wmparc.DKTatlas.mapped",
+        desc="White matter parcellation statistics file",
+    )
+    aparc_stats = OutputMultiPath(
+        File(exists=True),
+        loc="stats",
+        altkey="aparc.DKTatlas.mapped",
+        desc="Aparc parcellation statistics files",
+    )
+    curv_stats = OutputMultiPath(
+        File(exists=True),
+        loc="stats",
+        altkey="curv",
+        desc="Curvature statistics files"
+    )
+    w_g_stats = OutputMultiPath(
+        File(exists=True),
+        loc="stats",
+        altkey="w*g.pct",
+        desc="White minus gray statistics files"
+    )
+    aseg_presurf_stats = OutputMultiPath(
+        File(exists=True),
+        loc="stats",
+        altkey="aseg.presurf.hypos",
+        desc="Automated segmentation pre-surface recon statistics files"
+    )
+    sd = Directory(
+        exists=True,
+        argstr="--sd %s",
+        desc="Subjects directory"
+    )
+    sid = traits.String(
+        exists=True,
+        argstr="--sid %s",
+        desc="Subject ID"
+    )
+
+
```

Comment:
Is there anything in these beyond what's in `FSSourceOutputSpec`? If so, I would sub-class, e.g.:

```Python
class FastSurferSourceOutputSpec(FSSourceOutputSpec):
    additional_field = OutputMultiPath(File(exists=True), ...)
    ...
```